### PR TITLE
Update membership tier titles and descriptions

### PR DESCRIPTION
**Pull request by valentine**
Original text: 
> For discussion at the [2020-11-11 plenum](https://github.com/hackerspacesg/hackerspace.sg/blob/master/contents/plenum/2020-11-11/_index/main.md).

[Link to screen capture of changed content.](https://user-images.githubusercontent.com/676061/115938797-70588600-a4ce-11eb-8c40-9cb3fd9bcb9e.png)  (3.2MB PNG)

# Background

HackerspaceSG has not changed its membership tiers much since its inception.

Our running costs are around **$4,600 a month**, and our current layout allows for around ~17 desks at peak. This means that each desk costs roughly $270 a month, and any membership rate that is significantly lower is effectively a subsidised rate, including the standard $128/month tier.

As we are currently losing around **$1,500 a month**, it is high time for us to review our membership tiers, and see if we can align usage with contributions.

# Goals

- Make HackerspaceSG's current financial situation better.
- Better align the usage of resources (desk space, floor space, time in space) with contributions (both financial and in-kind).
- Encourage members with lower financial contributions to contribute other forms of effort towards maintaining and promoting the space.
  - Develop a sense of ownership in more members (see: the [IKEA effect](https://en.wikipedia.org/wiki/IKEA_effect)).
- Nudge members who are more hands-off towards contribute more financially, especially if they require almost daily use of the space.
- Continue to be financially inclusive to people who cannot afford it.
  - Create opportunities to contribute for people who are cash-poor, but time-rich. (The opportunities should hopefully align with their interests, and help build either their resumes or networks.)
- Remove unrealistic membership plans, and the potential confusion that the lower contribution tiers might be membership plans.

# Proposed changes

- Most significant change: **$128/month** (our most common offering) is now our Community tier, and allows for _~9 hours a week_ (roughly 1 day a week, or ~3 days of hanging out in the evenings) of space usage.
  - Most suitable for: occasional co-workers; people who come in after work or school to hang out.
- **$256/month** will be our Co-working tier, allowing for daily, _near unlimited_ use of the space. (Members are still not allowed to live out of the space, as we are not a licensed accommodation provider.)
  - Most suitable for: freelancers and small companies using the space daily during regular business hours; members who frequently spend the night at the space.
- **$64/month** is our Concessionary tier, for people who self-profess to be earning under a reasonable amount expected of technical professionals (currently proposed at $2,500/month).
  - Most suitable for: full-time students, NSFs, underemployed, unemployed.
  - (The current Students and NSFs categories will be removed — it is assumed that full-time students and NSFs will be included under the income criteria. It doesn't make sense for professionals taking on part-time degrees to be eligible for concessionary rates.)
- Opportunities for time/effort contributions in-lieu of money: members at the **$64/month** and **$128/month** tier will still be able to use the space for more than 9 hours a week, as long as they contribute \[30 minutes for $128; 60 minutes for $64] per 8 additional hours towards HSG "duties" (cleaning up after others, or doing pro-space activities like showing guests around or hosting events from the larger tech community).
  - One ancillary goal is to develop the abovementioned stronger sense of ownership and giving back in HSG members. It is ideal if most members give back more than they take.
- No proposed changes to the **$512/month** Dedicated Desk tier, but we might have to demarcate the "dedicated" area on our floor plan to prevent storage overflow.
- The **$1,337/month** tier will be removed. We may look at incorporating a "Talk to Us" section for potential generous benefactors.
- The **$16/month** and **$32/month** tiers will be moved to a separate Contribute (exact term to be worked on) section, to avoid confusion from potential and current members.

### CODE REVIEW

**Review by rolandturner — APPROVED** [review]
This looks good.

I remain not a fan of the <$64 tiers. If we were banking significant amounts of cash on them then, sure, but otherwise I fear that they anchor incorrectly. There's a clear reason for the $64 tier to exist, but it would seem to me to be sensible for it to be the lowest-cost option and for this fact to be obvious.

**Comment by valentine** [comment]
@rolandturner I've updated the descriptions further to reflect my proposal following our offline discussion.

I agree with you on the <$64 tiers, and we can table their removal or renaming at the January plenum. I believe that the spirit of it is to allow the community to contribute what they can, but it makes sense to split them out into a separate "Support Us" subsection to avoid giving off an impression that people are able to join as members at those rates.

File: contents/membership/_index/main.markdown [hunk 1]
```diff
@@ -78,9 +70,9 @@ subscription.
 <input type="hidden" name="hosted_button_id" value="6YY74YT8UKT9E">
 <table>
 <tr><td><input type="hidden" name="on0" value="HackerspaceSG monthly membership">HackerspaceSG monthly membership</td></tr><tr><td><select name="os0">
-	<option value="Regular">Regular : $128.00 SGD - monthly</option>
-	<option value="Associate">Associate : $64.00 SGD - monthly</option>
-	<option value="Resident">Resident : $256.00 SGD - monthly</option>
+	<option value="Resident">Regular : $256.00 SGD - monthly</option>
```
Comment (icedwater) [inline contents/membership/_index/main.markdown:73]: Suggest changing option value to Regular for sanity.

File: contents/membership/_index/main.markdown [hunk 2]
```diff
@@ -78,9 +70,9 @@ subscription.
...
+	<option value="Regular">Community : $128.00 SGD - monthly</option>
```
Comment (icedwater) [inline contents/membership/_index/main.markdown:74]: Suggest changing option value to Community for sanity.

File: contents/membership/_index/main.markdown [hunk 3]
```diff
@@ -78,9 +70,9 @@ subscription.
...
+	<option value="Associate">Concessionary : $64.00 SGD - monthly</option>
```
Comment (icedwater) [inline contents/membership/_index/main.markdown:75]: Why have the option values dissociated from the text?

**Review by icedwater — COMMENTED** [review]
Let's keep things consistent, first to avoid ambiguity over the "Regular" tier and second to keep option values and visible text coherent.

File: contents/membership/_index/main.markdown (hunk 3 above)
Comment (hyperair) [reply contents/membership/_index/main.markdown:75]: The option values match the Paypal subscription names, I think

**Comment by dyanawu** [comment]
I'd recommend switching this entire thing over into a feature matrix format to ensure the same information is available for all tiers; a paragraph each is not a great way to have comparisons now that there's extra discrete information relevant to each tier.

File: contents/membership/_index/main.markdown (hunk 3 above)
Comment (valentine) [reply contents/membership/_index/main.markdown:75]: What Bob said. I believe PayPal knows how much to charge for each subscription based on the value field, so they remain unchanged for now.

**Pull request merged**